.github/workflows/buildAndTest.yml

```
@@ -127,6 +127,25 @@ jobs:
         env:
           LD_LIBRARY_PATH: ${{ github.workspace }}/aienginev2/install/lib
 
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      # Build the repo test target in release mode to build and test.
+      - name: Build and test Python wheel
+        run: |
+          cp -R build_release/install/python/air python/wheel
```

```suggestion
          cp --archive build_release/install/python/air python/wheel
```
I tend to prefer this by keeping more stuff, including links for example. But of course it depends on the use case.